client/middleware.js

```diff
@@ -7,6 +7,7 @@ function go() {
   value = escape(value);
 
   localStorage.setItem("boardName", value);
+  patchBoardName("74171dcb-ee89-496a-828a-1b1c7302f628", "I am a small board")
```

Comment:
why is this hard coded

Reply:
The function works on it, right now we worked on the middleware first so to test the middleware we hardcoded it. When the frontend dev actually integrates these functions with the UI activity they will not be hardcoded